modules/reference/pages/properties/cluster-properties.adoc

```diff
@@ -1198,13 +1198,19 @@ Maximum number of bytes returned in a fetch request.
 
 The strategy used to fulfill fetch requests.
 
+* `polling`: Repeatedly polls every partition in the request for new data. The polling interval is set by <<fetch_reads_debounce_timeout,`fetch_reads_debounce_timeout`>> (deprecated).
+
+* `non_polling`: The backend is signaled when a partition has new data, so it doesn’t need to repeatedly read from every partition in the fetch. Redpanda recommends setting this value for most workloads, as it can improve fetch latency and CPU utilization.
```

Comment:
```suggestion
* `non_polling`: The backend is signaled when a partition has new data, so Redpanda doesn’t need to repeatedly read from every partition in the fetch. Redpanda Data recommends using this value for most workloads, because it can improve fetch latency and CPU utilization.
```